contracts/test/MockERC4626.sol

```diff
@@ -207,4 +211,27 @@ contract MockERC4626 is ERC4626, MultiRolesAuthority {
         );
         return accrued;
     }
+
```

Comment:
Why don’t we just update the other function to have a timestamp parameter?

Reply:
Yeah I guess we could huh, I was trying to keep this PR footprint small.  Ok let me do that instead.

Reply:
done.